packages/core/content-manager/server/src/history/content-types/history-version.ts

```diff
@@ -0,0 +1,63 @@
+import type { Schema } from '@strapi/types';
+
+const historyVersion: { schema: Schema.CollectionType } = {
+  schema: {
+    uid: 'plugin::content-manager.history-version',
+    modelName: 'history-version',
+    globalId: 'ContentManagerHistoryVersion',
+    kind: 'collectionType',
+    modelType: 'contentType',
+    collectionName: 'strapi_history_versions',
+    info: {
+      singularName: 'history-version',
+      pluralName: 'history-versions',
+      displayName: 'History Version',
+    },
+    pluginOptions: {
+      'content-manager': {
+        visible: false,
+      },
+      'content-type-builder': {
+        visible: false,
+      },
+      i18n: {
+        localized: false,
+      },
+    },
+    attributes: {
+      contentType: {
+        type: 'string',
+        required: true,
+      },
+      // documentId is a reserved attribute name
+      relatedDocumentId: {
+        type: 'string',
+        required: true,
+      },
+      locale: {
+        type: 'string',
+        required: false,
+      },
+      status: {
+        type: 'enumeration',
+        enum: ['draft', 'published', 'modified'],
+      },
+      data: {
+        type: 'json',
+      },
+      schema: {
+        type: 'json',
+      },
+      createdAt: {
+        type: 'date',
+      },
+      createdBy: {
+        type: 'relation',
+        relation: 'oneToOne',
+        target: 'admin::user',
+      },
```

Comment:
You don't get these attributes by default? (with updatedAt)

Reply:
Right now, maybe we do, I'm not sure since it looks like `options.timestamps` doesn't exist anymore.

But we're only using a content type temporarily, Alex will soon introduce a new v5 to register db models, so that's what we'll use. And for models, there will be no auto generated fields so we'll need to define them ourselves